pkg/operation/cloudbotanist/awsbotanist/addons.go

```diff
@@ -151,6 +151,9 @@ func (b *AWSBotanist) GenerateNginxIngressConfig() (map[string]interface{}, erro
 			"config": map[string]interface{}{
 				"use-proxy-protocol": "true",
 			},
+			"service": map[string]interface{}{
```

Comment:
As transporting the `loadBalancerSourceRanges` to the Helm chart seems to be not cloud specific, please move this part to https://github.com/gardener/gardener/blob/master/pkg/operation/hybridbotanist/addons.go#L147 and keep the cloud specific implementations untouched.